sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,47 @@
 Version history
 ===============
 
+6.6.1 (2021 March)
+------------------
+
+File format fixes and improvements:
+
+* CV7000
+   - fixed the channel-wise ordering of planes. The implementation changes to channel mapping 
+     will require existing memo files to be regenerated for CV7000 data that has multiple actions
```

Comment:
```suggestion
     will require existing memo files to be regenerated for CV7000 datasets
```

The regeneration will affect all CV7000 datasets since the internal class of the reader has been modified